provider/core/src/datagen/mod.rs

```diff
@@ -242,4 +242,8 @@ impl DataExporter for MultiExporter {
     fn close(&mut self) -> Result<(), DataError> {
         self.0.iter_mut().try_for_each(|e| e.close())
     }
+
+    fn supports_built_in_fallback(&self) -> bool {
```

Comment:
@sffc once again I've wasted 10 minutes because datagen behaves differently with different exporters

Reply:
We should make `supports_built_in_fallback` be non-default implemented